Dockerfile

```diff
@@ -34,12 +32,11 @@ ADD Gemfile /data/Gemfile
 ADD Gemfile.lock /data/Gemfile.lock
 RUN mkdir /data/build
 
-ARG RAILS_ENV=development
+#ARG RAILS_ENV=development
```

Comment:
If this is commented out, will we need to keep this?

Reply:
it's not _needed_ per se. I commented it out because at one point it was running in `development` mode in CCI, so I unset it in the `Dockerfile` in case that was why it wasn't obeying the value being set in the circle config.yml

Everywhere that this runs (docker-compose for dev, the `monit` infra for staging/prod) is forcing the RAILS_ENV value to however it's being run. 

I would rather have to explicitly set `RAILS_ENV` in deployments and have it fail if you don't set it than have it set a default that might be wrong and potentially cause weird behavior